pkgs/development/python-modules/deep_merge/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, fetchPypi, pytestCheckHook, nose }:
+
+buildPythonPackage rec {
+  pname = "deep_merge";
+  version = "0.0.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "tUQV+Qk0xC4zQRTihky01OczWzStOW41rYYQyWBlpH4=";
+  };
+
+  propagatedBuildInputs = [
+    nose
+  ];
+
+  doCheck = false;
```

Comment:
Please add a pythonImportsCheck.

Reply:
When added pythonImportsCheck it is always failing: 

```bash
deep_merge.tests.test_merge.test_default_behaviors ... ok
deep_merge.tests.test_merge.test_big_merge ... FAIL
deep_merge.tests.test_merge.test_no_references
Ensure that read-only parameters aren't being modified. ... FAIL

======================================================================
FAIL: deep_merge.tests.test_merge.test_big_merge
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/nix/store/g80x8dysb6hnhrcjrpav7dmsb8jnwnyr-python3.8-nose-1.3.7/lib/python3.8/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/private/tmp/nix-build-python3.8-deep_merge-0.0.4.drv-0/deep_merge-0.0.4/deep_merge/tests/test_merge.py", line 60, in test_big_merge
    assert merge(a, b, c) == expected
AssertionError

======================================================================
FAIL: deep_merge.tests.test_merge.test_no_references
Ensure that read-only parameters aren't being modified.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/nix/store/g80x8dysb6hnhrcjrpav7dmsb8jnwnyr-python3.8-nose-1.3.7/lib/python3.8/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/private/tmp/nix-build-python3.8-deep_merge-0.0.4.drv-0/deep_merge-0.0.4/deep_merge/tests/test_merge.py", line 73, in test_no_references
    assert output == {"key1": {"key1a": "2"}, "key2": [3, 4, 4, 5]}
AssertionError

----------------------------------------------------------------------
Ran 3 tests in 0.013s

FAILED (failures=2)
Test failed: <unittest.runner.TextTestResult run=3 errors=0 failures=2>
error: Test failed: <unittest.runner.TextTestResult run=3 errors=0 failures=2>
error: --- Error ------------------------------------------------------------------------------------------------------------------------ nix-build
builder for '/nix/store/41dyki3vl4qp53vd6kl92lg1yrn2w797-python3.8-deep_merge-0.0.4.drv' failed with exit code 1; last 10 log lines:
    File "/private/tmp/nix-build-python3.8-deep_merge-0.0.4.drv-0/deep_merge-0.0.4/deep_merge/tests/test_merge.py", line 73, in test_no_references
      assert output == {"key1": {"key1a": "2"}, "key2": [3, 4, 4, 5]}
  AssertionError

  ----------------------------------------------------------------------
  Ran 3 tests in 0.013s

  FAILED (failures=2)
  Test failed: <unittest.runner.TextTestResult run=3 errors=0 failures=2>
  error: Test failed: <unittest.runner.TextTestResult run=3 errors=0 failures=2>
```

Reply:
Hum, that does not look like a spurious failure. Can you open an issue upstream to ensure that it's not nixpkgs-specific?

Reply:
That is a 3 years old repo... But I will try.